.github/workflows/pending-prs.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Notify of Pending PRs'
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Generate Pending PRs list
+        id: generate-prs
+        run: |
+          ./.github/workflows/scripts/pending-prs slack redpanda-data/redpanda-operator > payload.json
+          echo "has-prs=$(cat payload.json | wc -l)" >> $GITHUB_OUTPUT
```

Comment:
This is basically a "did we actually have anything to post?" check. The `wc -l` will return `0` if there's nothing open or `1` if we have a payload to post.